tests/src/Integration/Action/UserRoleRemoveTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\rules\Integration\Action\UserRoleRemoveTest.
+ */
+
+namespace Drupal\Tests\rules\Integration\Action;
+
+use Drupal\Tests\rules\Integration\RulesEntityIntegrationTestBase;
+use Drupal\Tests\rules\Integration\RulesUserIntegrationTestTrait;
+
+/**
+ * @coversDefaultClass \Drupal\rules\Plugin\RulesAction\UserRoleRemove
+ * @group rules_actions
+ */
+class UserRoleRemoveTest extends RulesEntityIntegrationTestBase {
+
+  use RulesUserIntegrationTestTrait;
```

Comment:
That trait does not exist anymore, can you merge in the 8.x-3.x branch and converted the test to use Prophecy mocking?